app/forms/journeys/additional_payments_for_teaching/claim_submission_form.rb

```diff
@@ -0,0 +1,53 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class ClaimSubmissionForm < ::ClaimSubmissionBaseForm
+      private
+
+      def selected_claim_policy
+        case answers.selected_policy
+        when "EarlyCareerPayments"
+          Policies::EarlyCareerPayments
+        when "LevellingUpPremiumPayments"
+          Policies::LevellingUpPremiumPayments
+        when nil
+          nil
+        else
+          fail "Invalid policy selected: #{answers.selected_policy}"
+        end
+      end
+
+      def main_eligibility
+        @main_eligibility ||= eligibilities.detect { |e| e.policy == main_policy }
+      end
+
+      def main_policy
+        if selected_claim_policy.present?
+          selected_claim_policy
+        else
+          Policies::EarlyCareerPayments
+        end
+      end
+
+      def calculate_award_amount(eligibility)
+        eligibility.award_amount = eligibility.calculate_award_amount
+      end
+
+      def generate_policy_options_provided
+        eligible_now_and_sorted.map do |e|
+          {
+            "policy" => e.policy.to_s,
+            "award_amount" => BigDecimal(e.award_amount)
+          }
+        end
+      end
+
+      def eligible_now_and_sorted
+        eligible_now.sort_by { |e| [-e.award_amount.to_i, e.policy.short_name] }
+      end
+
+      def eligible_now
+        eligibilities.select { |e| e.status == :eligible_now }
+      end
```

Comment:
I think we need both as we use both eligibilities to generate the eligibility attributes stored on the claim, at least according to [some of the specs](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/c2bc9465af9f70830b60c110d3da8171755839f4/spec/features/early_career_payments_claim_spec.rb#L279). Not sure if this is something that we care about in practice.

Reply:
Looks like it is shown in the admin so they know what options were provided during the journey. This probably belongs in the journey session object rather than the claim. I'm happy to ignore this for now if we create a new ticket to address this later.